<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main</class>
 <widget class="QWidget" name="main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>264</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QuickAi快问Ai</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>271</width>
     <height>251</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>主页</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>261</width>
       <height>201</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #6BC6FF, stop:1 #0077FF);
    border-style: solid;
    border-radius: 5px;
    border-width: 2px;
    border-color: #005D9F;
    color: white;
    padding: 10px;
}

QPushButton:hover {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #7EDFFF, stop:1 #0088FF);
}

QPushButton:pressed {
    background-color: #005D9F;
    border-color: #003366;
}</string>
        </property>
        <property name="text">
         <string>打开Ai大模型官网</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_11">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #7CFC00, stop:1 #32CD32);
    border-style: solid;
    border-radius: 5px;
    border-width: 2px;
    border-color: #006400;
    color: white;
    padding: 10px;
}

QPushButton:hover {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #ADFF2F, stop:1 #7FFF00);
}

QPushButton:pressed {
    background-color: #006400;
    border-color: #003300;
}</string>
        </property>
        <property name="text">
         <string>打开官网</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #FF6B6B, stop:1 #FF0077);
    border-style: solid;
    border-radius: 5px;
    border-width: 2px;
    border-color: #9F005D;
    color: white;
    padding: 10px;
}

QPushButton:hover {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #FF7E7E, stop:1 #FF0088);
}

QPushButton:pressed {
    background-color: #9F005D;
    border-color: #660033;
}
</string>
        </property>
        <property name="text">
         <string>打开教程</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_7">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #FFA500, stop:1 #FF7F00);
    border-style: solid;
    border-radius: 5px;
    border-width: 2px;
    border-color: #8B4500;
    color: white;
    padding: 10px;
}

QPushButton:hover {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #FFBF00, stop:1 #FF8800);
}

QPushButton:pressed {
    background-color: #8B4500;
    border-color: #663300;
}</string>
        </property>
        <property name="text">
         <string>购买正式版</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>200</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>检测到新版本请去官网下载最新版本</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>设置</string>
    </attribute>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>261</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0">
      <item>
       <widget class="QListView" name="listView"/>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="tableView"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>261</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QCheckBox" name="checkBox_2">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>悬浮窗跟随</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>悬浮操作条</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBox_3">
        <property name="text">
         <string>打开至托盘</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="checkBox_4">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>开机自启动</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,2,1">
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>ChatGpt</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>通义千问</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>文心一言</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>讯飞星火</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>智谱清言</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>字节豆包</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>deepseek</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>kimi</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>mytan</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>模型选择</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>打开Ai官网</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>comboBox</sender>
   <signal>activated(int)</signal>
   <receiver>main</receiver>
   <slot>model()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>main</receiver>
   <slot>popup()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>83</x>
     <y>500</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_2</sender>
   <signal>stateChanged(int)</signal>
   <receiver>main</receiver>
   <slot>popups()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_3</sender>
   <signal>stateChanged(int)</signal>
   <receiver>main</receiver>
   <slot>tray()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>429</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_4</sender>
   <signal>stateChanged(int)</signal>
   <receiver>main</receiver>
   <slot>start()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>371</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listView</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>main</receiver>
   <slot>category_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>93</x>
     <y>457</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>main</receiver>
   <slot>ai()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>main</receiver>
   <slot>ai()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_11</sender>
   <signal>clicked()</signal>
   <receiver>main</receiver>
   <slot>ztough()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>437</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>main</receiver>
   <slot>tutorial()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>438</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked()</signal>
   <receiver>main</receiver>
   <slot>renew()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>466</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>model()</slot>
  <slot>popup()</slot>
  <slot>popups()</slot>
  <slot>tray()</slot>
  <slot>start()</slot>
  <slot>category_click()</slot>
  <slot>save()</slot>
  <slot>ai()</slot>
  <slot>ztough()</slot>
  <slot>tutorial()</slot>
  <slot>renew()</slot>
 </slots>
</ui>
